modules/experimental-layers/src/mesh-layer/mesh-layer-vertex.glsl.js

```diff
@@ -12,43 +12,27 @@ attribute vec2 texCoords;
 // Instance attributes
 attribute vec3 instancePositions;
 attribute vec2 instancePositions64xy;
-attribute vec3 instanceRotations;
 attribute vec4 instanceColors;
 attribute vec3 instancePickingColors;
+attribute vec4 instanceModelMatCol1;
```

Comment:
Nit: Type out `instanceModelMatrixColumn1`. I realize this identifier is getting long, but I do think it is worth typing things out, and the existing code base is quite consistent in this regard. 

(Anything prefaced with "Nit:" means that this is a suggestion that you can ignore without further discussion if you really prefer it the way you wrote it.)